components/brave_perf_predictor/browser/third_party_extractor.cc

```diff
@@ -0,0 +1,116 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_perf_predictor/browser/third_party_extractor.h"
+
+#include "base/logging.h"
+#include "base/values.h"
+#include "third_party/re2/src/re2/re2.h"
+#include "rapidjson/document.h"
```

Comment:
didn't notice this the first time around, but we definitely do not want to use rapidjson for anything. Please convert to base JSONReader/JSONWriter